integration_test.go

```diff
@@ -379,6 +379,18 @@ func TestIntegration(t *testing.T) {
 				{"php/crappy.php", time.Date(2015, time.April, 5, 21, 30, 47, 0, time.UTC)},
 			},
 		},
+		{
+			`
+				SELECT repository_id, JSON_EXTRACT(bl, "$.author[0]"),
+					   ARRAY_LENGTH(JSON_EXTRACT(bl, "$.file"))
```

Comment:
once blame is a struct this will become:

```sql
SELECT repository_id, JSON_EXTRACT(bl.author, "$[0]"),
					   ARRAY_LENGTH(bl.file)
```

Reply:
structs weren't merged, right?

Reply:
Nopes, so it's ok for now